app/views/layouts/error.html.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="<%= t("sufia.document_language") %>">
```

Comment:
These localization references need to be curation_concerns.  Also, I couldn't find the localization string document_language.